R/lint.R

```diff
@@ -17,7 +17,7 @@
 #' @param ... Provide additional arguments to be passed to:
 #'   - [exclude()] (in case of `lint()`; e.g. `lints` or `exclusions`)
 #'   - [lint()] (in case of `lint_dir()` and `lint_package()`; e.g. `linters` or `cache`)
-#' @param cache When logical, toggle caching of lint results. I1f passed a character string, store the cache in this
+#' @param cache When logical, toggle caching of lint results. If passed a character string, store the cache in this
```

Comment:
this needs to be reflected in the .Rd file as well

Reply:
Aight. I've also included the changes from https://github.com/r-lib/lintr/pull/2268.